cern_access/indico_cern_access/util.py

```diff
@@ -194,7 +197,8 @@ def withdraw_event_access_request(req):
 
 def get_random_reservation_code():
     """Generate random reservation code for data required by ADaMS API."""
-    return 'I' + ''.join(random.sample(string.ascii_uppercase.replace('O', '') + string.digits, 6))
+    charset = string.ascii_uppercase.replace('O', '').replace('0', '') + string.digits
```

Comment:
Should we get rid of I/1 as well?

Reply:
I don't think those are as misleading. Most fonts handle them well.

Reply:
`0` is in `string.digits`, not `string.ascii_uppercase`

Reply:
maybe the whole thing is more readable if we just put a string `'ABC...Z123456789'` there?